src/graphics/program-lib/programs/standard.js

```diff
@@ -465,6 +467,8 @@ var standard = {
         var needsNormal = lighting || reflections || options.ambientSH || options.prefilteredCubemap || options.heightMap || options.enableGGXSpecular;
         var shadowPass = options.pass >= SHADER_SHADOW && options.pass <= 17;
 
+        needsNormal = needsNormal || graphCode;
```

Comment:
does it always need normal?

Reply:
technically it doesn't - so I will add a check if the shader graph needs normal

Reply:
updated